src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private final static Scanner SCANNER = new Scanner(System.in);
```

Comment:
접근 제어자 순서에도 [컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)이 있습니다 😃 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```

Reply:
넵! 컨벤션 확인해보고 사용하겠습니다!